README.md

```diff
@@ -10,41 +10,79 @@
 
 # Temporary Note
 
-This project is not ready for deployment and the documentation may be incorrect.
+  Changes are made on a daily basis to documentation and files. Documentation may not be accurate due to the rapid changes that are occuring. In its current state, this repo is not ready for deployment.
 
 # Welcome!
 
 This is incoming v2 front-end application for [OperationCode](https://operationcode.org). We highly recommend [joining our organization](https://operationcode.org/join) to receive an invite to our Slack team. From there, you'll want to join the `#oc-projects` channel. You can get help from multiple professional developers, including people who have worked on the application since day 1!
 
-Before contributing, please review our [GitHub Workflow]().
+Before contributing, please review our [Contributing Guide](CONTRIBUTING.md)
 
-## Maintainers
+# Maintainers
 
 For information about the maintainers of the project, check out [MAINTAINERS.md](MAINTAINERS.md).
 
-## Quick Start
-
+# Quick Start
 If you're unsure of how to start this app or code for it, don't worry! You're our target audience!
 Please read our [Contributing Guide](CONTRIBUTING.md) to learn everything you need to be able to ask the right questions on our Slack team.
 
 If you're no stranger to React applications and the JavaScript ecosystem:
 
-Version Assumptions:
-[Click to see Node version](https://github.com/OperationCode/front-end/blob/master/.nvmrc)
-yarn@1.7.0
+Recommended Versions of tools used within the repo:
+Node @ 8.11.3 LTS
+Yarn@1.7.0 or greater
+Git @2.17.1 or greater
+
+## Github workflow
+
+Fork a copy to your Github profile
 
 ```sh
-# Clone the repo
-git clone git@github.com:OperationCode/front-end.git
+# Clone your fork of the repo
+git clone https://git@github.com/yourusername/front-end.git
+
+#set upstream to receive all the latest commits and keep your local repo in sync
+
+git remote add upstream https://github.com/OperationCode/front-end.git
+
+#checkout a branch with the name of the issue
+git checkout -b "name-of-branch-relevant-to-issue"
 
+```
+When finished, [create a PR on GitHub](https://help.github.com/articles/creating-a-pull-request/)
+
+## Development workflow
+
+```sh
 # Install dependencies
 yarn
 
-# Run local development
+# Run local development on localhost:3000
 yarn dev
-# open localhost:3000
 
 # Use Storybook when developing new common components or to see what lego blocks you have available
 yarn storybook
 # open localhost:9001
-```
+
+#Create a static bundle of our Storybook instance that can be easily deployed.
+yarn build-storybook
```

Comment:
I think it's useful to describe what every script does, but I don't believe it belongs in `#Quick Start`. Moving these to `CONTRIBUTING.md` is a good call.